build.mk

```diff
@@ -51,7 +51,7 @@ distclean: distclean-local
 .PHONY: check
 check:
 	@echo "   [check]"
-	@cppcheck $(CPPFLAGS) $(CHECKOPTS) src
+	@cppcheck $(CPPFLAGS) $(CHECKOPTS) $(SRC) -I$(INC)
```

Comment:
`-I$(INC)` is in `$(CPPFLAGS)` as per the `config.mk` definition (l.21). So it should not be repeated here... 

The use of `src` instead of `$(SRC)` was intentional here for the `test` compilation, but it's no longer relevant there, so $(SRC) should be fine if it does not trip up the CI...